<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D2BlockClass</class>
 <widget class="QMainWindow" name="D2BlockClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>316</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>D2Block</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/d2block.qrc">
    <normaloff>:/D2Block/d2block.ico</normaloff>:/D2Block/d2block.ico</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="progressTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>491</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Updating Diablo II Ignorelist...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filePath">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G:/Games/Diablo II/ignorelist</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/d2block.qrc"/>
 </resources>
 <connections/>
</ui>
